tests/trainer/logging_/test_distributed_logging.py

```diff
@@ -23,23 +23,53 @@
 from tests.helpers.runif import RunIf
 
 
+class AllRankLogger(LightningLoggerBase):
```

Comment:
above here, need to remove the unused import. 

I couldn't do it due to permission error pushing to your branch.